erpnext/accounts/doctype/payment_entry/payment_entry.json

```diff
@@ -587,6 +619,38 @@
    "translatable": 0, 
    "unique": 0
   }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "cost_center_to", 
```

Comment:
Any particular reason to have two fields, cost_center_from and cost_center_to fields?

Reply:
Yes, this is for if user want to do internal transfer from one cost center to another.

Reply:
why in payment entry? since its an exception workflow they can move it via a journal entry (?)